docs/conandata_yml_format.md

```diff
@@ -63,85 +63,79 @@ Sometimes it is useful to declare mirrors, use a list in the `url` field. Conan
 
-[sha1](https://tools.ietf.org/html/rfc3174) is an alternate method to specify hash sum. It's usage is discouraged and `sha256` is preferred.
-
-#### md5
-
-[md5](https://tools.ietf.org/html/rfc1321) is an alternate method to specify hash sum. It's usage is discouraged and `sha256` is preferred.
-
-### Other cases
-
-There are other ways to specify sources to cover other cases.
-
-#### Source code & license
-
-Certain projects provide license on their own, and released artifacts do not include it. In this case, a license URL can be provided separately:
-
-```
+```yml
 sources:
-  8.0.0:
-    - url: https://github.com/approvals/ApprovalTests.cpp/releases/download/v.8.0.0/ApprovalTests.v.8.0.0.hpp
-      sha256: e16a97081f8582be951d95a9d53dc611f1f5a84e117a477029890d0b34ae99d6
-    - url: "https://raw.githubusercontent.com/approvals/ApprovalTests.cpp/v.8.0.0/LICENSE"
+  "10.12.2":
+    "sources":
+      url: https://github.com/approvals/ApprovalTests.cpp/releases/download/v.10.12.2/ApprovalTests.v.10.12.2.hpp
+      sha256: 4c43d0ea98669e3d6fbb5810cc47b19adaf88cabb1421b488aa306b08c434131
+    "license":
+      url: "https://raw.githubusercontent.com/approvals/ApprovalTests.cpp/v.10.12.2/LICENSE"
       sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
 ```
 
-#### Several source code archives
+> **Note**: This is the way the linter accepts
 
-Some projects may include multiple tarballs as a part of release, [OpenCV](https://opencv.org/) is an example which includes auxiliary [contrib](https://github.com/opencv/opencv_contrib) archive:
+You can list as many assets you need and reference them by their index. But make sure you keep them in order if there is any specific
+logic about handling.
 
-```
+```yml
 sources:
-  "4.5.0":
-    - sha256: dde4bf8d6639a5d3fe34d5515eab4a15669ded609a1d622350c7ff20dace1907
-      url: https://github.com/opencv/opencv/archive/4.5.0.tar.gz
-    - sha256: a65f1f0b98b2c720abbf122c502044d11f427a43212d85d8d2402d7a6339edda
-      url: https://github.com/opencv/opencv_contrib/archive/4.5.0.tar.gz
+  "10.12.2":
+    - url: https://github.com/approvals/ApprovalTests.cpp/releases/download/v.10.12.2/ApprovalTests.v.10.12.2.hpp
+      sha256: 4c43d0ea98669e3d6fbb5810cc47b19adaf88cabb1421b488aa306b08c434131
+    - url: "https://raw.githubusercontent.com/approvals/ApprovalTests.cpp/v.10.12.2/LICENSE"
+      sha256: c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
 ```
-
 #### Different source code archives per configuration
 
-This is the most advanced and sophisticated use-case, but no so common. Some projects may provide different sources for different platforms for awkward reasons, it could be expressed as:
+This is the most advanced and sophisticated use-case, but not so common. Some projects may provide different sources for different platforms, it could be expressed as:
+
+> **Warning**: The Linter does not expect different compilers
```

Comment:
Only 1 recipe needs this https://github.com/conan-io/conan-center-index/blob/e907fe23f5ee099a4f3e78e848931a315e90aa30/recipes/cspice/all/conandata.yml